user/libeconf/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "libeconf"
+pkgver = "0.7.3"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "ninja", "pkgconf", "doxygen"]
+makedepends = []
+pkgdesc = "Config file parser"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "MIT"
+url = "https://github.com/openSUSE/libeconf"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "2e185c86674cdf128b518988cc740c27bdc93adafadf65cb6e3e860538fefefa"
+tool_flags = {"CFLAGS": ["-Wl,--undefined-version", "-D_FORTIFY_SOURCE=2"]}
```

Comment:
```suggestion
```

the fortify doesn't do anything, and the -Wl is ldflags not cflags, but use options = linkundefver for that

Reply:
if you want to silence the fortify warnings then delete https://github.com/openSUSE/libeconf/blob/d78b86e72b5818b93f06c54b8fb8e19669b45844/meson.build#L19 i guess